core/src/App.html

```diff
@@ -1388,7 +1388,14 @@
           e.data.data.id,
           e.data.data.operation,
           e.data.data.params
-        );
+        )
+      }
+
+      if('luigi-runtim-error-handling'===e.data.msg){
```

Comment:
```suggestion
      if('luigi-runtime-error-handling'===e.data.msg){
```
Typo?  Same as [luigi-client.js line20](https://github.com/SAP/luigi/pull/2167/files#diff-52086c5914c2bb232f321aef8650f2d284bc19d0052b6b222e704d47bece9755R20)?